source/administration/devops-command-center.rst

```diff
@@ -63,12 +63,20 @@ Mattermost Incident Management includes built-in slash commands:
 Adding slash commands to tasks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Slash commands can be added to tasks to initiate actions as part of your incident response playbook.
+Slash commands can be added to tasks to initiate actions as part of your playbook.
 
 Here are some examples:
 
-- Add a task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
-- One of your tasks may require the header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
+- Add a communication task called **Sync up** with the slash command ``/zoom hello``. Running that slash command initiates a Zoom call in the incident channel. If you've installed Jitsi, you could use ``/jitsi hello``. 
+- One of your tasks may require the channel header to be changed to reflect a new status. Create a task called **Change header** with the slash command ``/header new header``.
+
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
+
+You can use the test commands to create incidents that are populated with random data. These incidents are listed in the incident insight page.
+
+- ``/incident test bulk-data``: This command accepts a number of ongoing incidents, a number of ended incidents, a beginning and an end date, and an optional seed. It creates as many ongoing and ended incidents as specified, all of them with their creation date randomly picked between the beginning and end dates. The seed, if available, is used to get reproducible results. The names of the incidents are randomly chosen from a list of incident names and a list of fake company names which are defined in the code. An example command is: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``.
+- ``/incident test create-incident``: This command accepts a playbook ID (that can be chosen from the playbooks the user is a member of, using the autocomplete system), a timestamp, and an incident name. It creates an ongoing incident with the creation date set to the specified timestamp. An example command looks like this: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``.
```

Comment:
The examples here are swapped!

Reply:
Yep, @cwarnermm suggested swapping them which I did but I should have confirmed whether they're in a specific order? If they are, I will swap them around and add a note indicating that one needs to be used prior to the other.

Reply:
Good catch on order of operations for running the commands. In addition, the command examples appear to be for the opposite command where ``/incident test create-incident has an example for bulk-data and vice-versa.

Reply:
> the command examples appear to be for the opposite command where ``/incident test create-incident has an example for bulk-data and vice-versa

Oh yeah, that's what I meant, sorry for the confusion!